src/__snapshots__/selectors.spec.js.snap

```diff
@@ -239,51 +249,20 @@ exports[`selectors should retrieve initialized value 1`] = `true`;
 
 exports[`selectors should retrieve keys 1`] = `
 Array [
+  "0",
```

Comment:
Here 0 should not be the first item.

Reply:
It may be overkill , but the `Map` object in ES6 seems to guarantee the key/value order

Reply:
_notes_
 - Map.set(key, value)
Something to try:
```
state {
  data: Array.from(map.entries()),
  initialized: <bool>,
}
```
  
+ selectors + memoisation
`Array.from(new Map(state).values())`
`Array.from(new Map(state).keys())`